Apps/ValidationTests/Win32/Resource.h

```diff
@@ -0,0 +1,30 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
```

Comment:
Should this be checked in? We have it checked in in the other place, so I assume so, but just thought I'd check to be sure.

Reply:
I need that .h for icon, app strings,...